### ISSUE TITLE
DOSBox-X won't recognize axis 3 and 4 for joystick

### ISSUE THREAD

**Issue by Vitoria90** [joncampbell123/dosbox-x#5176]
### Describe the bug

Trying to play F-14 Fleet Defender, which support 4 axes of control. DBX detects the X and Y axes of my joystick, but doesn't detect Rz (stick twist) or the throttle Slider. The Mapper Tool can't seem to detect axes 3 and 4 either.

### Steps to reproduce the behaviour

1. Configure the joystick to 4axis, ensure vJoy is running and only the vJoy Device is visible to programs.
2. Launch the game, enter the joystick callibration mode
3. X and Y are recognized, the other two are not

Alternatively,

2. Open the Mapper Tool
3. X and Y are recognized and registered, axes 3 and 4 are not.

### What operating system(s) this bug have occurred on?

Windows 10 Pro

### What version(s) of DOSBox-X have this bug?

v. 2024.03.01 (VS XP SDL1 64-bit)

### Used configuration

```ini
https://pastebin.com/iiYNPBeJ
```

### Have you checked that no similar bug report(s) exist?

- [X] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [X] I agree to follow the code of conduct and the contributing guidelines.

**Comment by maron2000** [comment]
All 4 axis of my DualShock 2 is recognized on 2024.03.01VS XP x86, x64 SDL1.
(And 2024.07.01 standard x86, x64 SDL1 builds as well)

![image](https://github.com/user-attachments/assets/b9f1077c-5b65-4f81-bf02-6bc11b8549f3)

**Comment by Vitoria90** [comment]
So yesterday I hadn't thought of screencapping it, but my Mapper was actually showing Joystick 1 and 2, instead Axis 1/2 and 3/4, though I had made sure it was set to 4axis. Booting it up now it does show 1/2 and 3/4.
![image](https://github.com/user-attachments/assets/19c39e1a-3d2b-442b-bdea-fc670ea67736)

I guess I should've tried the age old trick of rebooting my whole computer, but it hadn't crossed my mind yesterday. I got two other MPS combat flight sims to configure here, if that happens again where it shows two joysticks on the mapper when I set it to 4axis I'll update this thread. Sorry for the false alarm.
